app/docs/dev/policies/meshproxypatch.md

```diff
@@ -0,0 +1,862 @@
+            listenerName: inbound:127.0.0.0:80 # optional: if absent, filter will be patched within all listeners regardless of name
+            listenerTags: # optional: if absent, filter will be patched within all listeners regardless of listener tags
+              kuma.io/service: backend
+            origin: inbound # optional: if absent, filter will be patched within all listeners regardless of its origin
+          value: | # you can specify only part of filter definition that will be merged into existing filter
+            name: envoy.filters.network.tcp_proxy
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+              idleTimeout: 10s
+      - networkFilter:
+          operation: Remove
+          match: # optional: if absent, all filters from all listeners will be removed
+            name: envoy.filters.network.tcp_proxy # optional: if absent, all filters regardless of name will be removed
+            listenerName: inbound:127.0.0.0:80 # optional: if absent, all filters regardless of the listener name will be removed
+            listenerTags: # optional: if absent, all filters regardless of the listener tags will be removed
+              kuma.io/service: backend
+            origin: inbound # optional: if absent, all filters regardless of its origin will be removed
+```
+{% endtab %}
+{% endtabs %}
+
+#### HTTP Filter
+
+Modifications that are applied on [HTTP Filters](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/filter/http/http) that are part of [Listeners](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto#config-listener-v3-listener) resource.
+Modifications that {{site.mesh_product_name}} applies on all [HTTP Connection Managers](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto.html#http-connection-manager-proto) in the Listener.
+
+HTTP Filter modifications can only be applied on services [configured as HTTP](/docs/{{ page.version }}/policies/protocol-support-in-kuma).
+
+Available operations:
+* `AddFirst` - add a new filter as a first filter in HTTP Connection Manager.
+* `AddLast` - add a new filter as a last filter in HTTP Connection Manager.
+* `AddAfter` - add a new filter after other filter in HTTP Connection Manager that is matched using `match` section.
+* `AddBefore` - add a new filter before other filter in HTTP Connection Manager that is matched using `match` section.
+* `Patch` - patch a matched filter in HTTP Connection Manager.
+* `Remove` - remove a filter in HTTP Connection Manager.
+
+Available matchers:
+* `name` - name of the network filter
+* `listenerName` - name of the listener
+* `listenerTags` - tags of inbound or outbound listeners. They match `Listener.metadata.filterMetadata[io.kuma.tags]` in XDS configuration.
+* `origin` - origin of the listener
```

Comment:
```suggestion
* `name` - name of the network filter.
* `listenerName` - name of the listener.
* `listenerTags` - tags of inbound or outbound listeners. They match `Listener.metadata.filterMetadata[io.kuma.tags]` in XDS configuration.
* `origin` - origin of the listener.
```
As other envoy related sections starts with capital letter, shouldn't we do the same here with `Listener`?